samples/expensive-test/create-data-labeling-job-image.test.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {assert} = require('chai');
+const {after, describe, it} = require('mocha');
+const uuid = require('uuid').v4;
+const cp = require('child_process');
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+const cwd = path.join(__dirname, '..');
+
+const displayName = `temp_create_data_labeling_job_image_test_${uuid()}`;
+const datasetId = '1905673553261363200';
+const instructionUri =
+  'gs://ucaip-sample-resources/images/datalabeling_instructions.pdf';
+const annotationSpec = 'roses';
+const project = process.env.CAIP_PROJECT_ID;
```

Comment:
So curiosity - why is this different than what is set in `GCLOUD_PROJECT`?

Reply:
It's the same now.